src/views/AboutWindow.qml

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright(c) 2024 Sven Trittler
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+*/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import org.eclipse.cyclonedds.insight
+
+
+Window {
+    id: aboutWindow
+
+    property int aboutWidth: 500
+    property int aboutHeight: 200
+
+    width: aboutWidth
+    height: aboutHeight
+    minimumWidth: aboutWidth
+    minimumHeight: aboutHeight
+    maximumWidth: aboutWidth
+    maximumHeight: aboutHeight
+
+    title: "About CycloneDDS Insight"
+    visible: false
+    flags: Qt.Dialog
+    modality: Qt.ApplicationModal
+
+    color: rootWindow.isDarkMode ? Constants.darkOverviewBackground : Constants.lightOverviewBackground
+
+    RowLayout {
+        anchors.fill: parent
+
+        Rectangle {
+            Layout.preferredWidth: parent.width * 0.33
+            Layout.fillHeight: true
+            color: rootWindow.isDarkMode ? Constants.darkOverviewBackground : Constants.lightOverviewBackground
+
+            Image {
+                source: "qrc:/res/images/cyclonedds.png"
+                anchors.centerIn: parent
+                fillMode: Image.PreserveAspectFit
+                width: parent.width * 0.8
+                height: parent.height * 0.8
+            }
+        }
+
+        Rectangle {
+            Layout.preferredWidth: parent.width * 0.67
+            Layout.fillHeight: true
+            color: rootWindow.isDarkMode ? Constants.darkHeaderBackground : Constants.lightHeaderBackground
+
+            Column {
+                anchors.centerIn: parent
+                spacing: 10
+
+                Label {
+                    text: "CycloneDDS Insight"
+                    font.pixelSize: 25
+                    font.bold: true
+                    horizontalAlignment: Text.AlignHCenter
+                }
+
+                Label {
+                    text: "Version: " + CYCLONEDDS_INSIGHT_VERSION
+                    font.pixelSize: 15
+                    horizontalAlignment: Text.AlignHCenter
+                }
+
+                Text {
+                    text: "© 2024 Eclipse Cyclone DDS™"
```

Comment:
I like the idea of the "About Window", but in my understanding the copyright notice is incorrect: The copyright doesn't belong to "Eclipse Cyclone DDS" because (1) "Eclipse Cyclone DDS" is a trademark, not a person or legal entity, and (2) the Eclipse Foundation leaves the copyright with the original owners.

I would replace either by a complete list of contributors to Insight (currently a rather short list!), on the grounds that this is just an application using the API of Cyclone; or to something like "Contributors to the Eclipse Cyclone DDS project", on the grounds that Insight is considered a part of Cyclone DDS and that listing everyone here is somewhat impractical and would moreover require updating the copyright window on contributions to other repositories.

Reply:
@eboasson thank you for the review, i totally agree. I updated the text, can you review again? :)

<img width="549" alt="Screenshot 2024-09-27 at 16 12 05" src="https://github.com/user-attachments/assets/2c2eea34-1337-4f41-9774-86f9d8d3f95d">